documentation/src/main/asciidoc/topics/attributes/transaction-attributes.adoc

```diff
@@ -5,7 +5,7 @@
 :tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/TransactionManagerLookup.html[TransactionManagerLookup]
 :embedded-tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/EmbeddedTransactionManagerLookup.html[EmbeddedTransactionManagerLookup]
 :jboss-tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/JBossStandaloneJTAManagerLookup.html[JBossStandaloneJTAManagerLookup]
-:wildfly-tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/WildflyTransactionManagerLookup.html[WildflyTransactionManagerLookup]
+:wildfly-tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/WildflyTransactionManagerLookup.html[WildFlyTransactionManagerLookup]
```

Comment:
the class name has a lower case `f`. 
this change can be removed.

Reply:
Oops. Let me fix that.

Reply:
Done (though the URL would still work haha).